src/main/perf/facets/IndexFacets.java

```diff
@@ -124,4 +131,21 @@ public static void main(String args[]) throws IOException {
         AtomicInteger counter = new AtomicInteger(0);
         return item -> consumer.accept(counter.getAndIncrement(), item);
     }
-}
\ No newline at end of file
+
+    private static void deleteDirectoryIfExists(Path path) throws IOException {
+        if (Files.exists(path)) {
+            try(Stream<Path> walk = Files.walk(path)) {
+                walk.sorted(Comparator.reverseOrder()).forEach(IndexFacets::deleteFile);
+            }
+        }
+    }
+
+    private static void deleteFile(Path path) {
+        try {
+            Files.delete(path);
+        } catch (IOException e) {
```

Comment:
If I rethrow the error here, I can't use this function in `#forEach()` (see line 138). I think printing the stack trace and exiting is an OK alternative.